scripts/cron/cron_run_jobs.php

```diff
@@ -177,7 +177,15 @@
 				exit(-1);
 			}
 			// Execute job
+			$oldstatus = $cronjob->status;
```

Comment:
There is already a status to know if job is processed or not inside the run_job method.
Also the status "running" or not must differ from the status the job is a draft or ok for run.